101-machine-learning-compute-create-computeinstance/azuredeploy.json

```diff
@@ -1,92 +1,67 @@
+              "description": "Specifies the name of the Azure Machine Learning compute instance to be deployed"
             }
         },        
         "location": {
-            "type": "string",
-            "defaultValue":"[resourceGroup().location]",
+            "type": "string",            
             "metadata": {
               "description": "Location of the Azure Machine Learning workspace."
             }
         },
-        "adminUserName": {
-            "type": "securestring",
-            "metadata": {
-              "description": "The admin username needed to access the virtual machine."
-            }
-        },
-        "adminPublicKey": {
-            "type": "securestring",
-            "metadata": {
-              "description": "The SSH public key of the administrator user account."
-            }
-        },
-        "sshPublicAccess": {
-            "defaultValue": "Enabled",
-            "allowedValues": [
-                "Enabled",
-                "Disabled"
-            ],
-            "type": "string",
-            "metadata": {
-              "description": "Indicates the state of the public SSH port. Eg Enabled/Disabled"
-            }
-        },
-        "sshPort": {
-            "defaultValue": 22,
-            "type": "int",
-            "metadata": {
-              "description": "The exposed port for the compute instance."
-            }
-        },
         "vmSize": {
             "defaultValue": "Standard_DS3_v2",
             "type": "string",
             "metadata": {
-              "description": "The size of agent VMs. More details can be found here: https://aka.ms/azureml-vm-details."
+              "description": "The VM size for compute instance"
             }
         },
         "vnetResourceGroupName": {
             "defaultValue": "",
             "type": "string",
             "metadata": {
-                "description": "Name of the resource group which holds the VNET to which you want to inject your compute in."
+                "description": "Name of the resource group which holds the VNET to which you want to inject your compute instance in."
             }
         },
         "vnetName": {
             "defaultValue": "",
             "type": "string",
             "metadata": {
-                "description": "Name of the vnet which you want to inject your compute in."
+                "description": "Name of the vnet which you want to inject your compute instance in."
             }
         },
         "subnetName": {
             "defaultValue": "",
             "type": "string",
             "metadata": {
-                "description": "Name of the subnet inside the VNET which you want to inject your compute in."
+                "description": "Name of the subnet inside the VNET which you want to inject your compute instance in."
             }
-        }
+        },
+        "tenantId": {            
```

Comment:
is this optional?

Reply:
No this is a required parameter for admin create on behalf of user capability

Reply:
add a defaultValue of subscription().tenantId